### ISSUE TITLE
Cannot find VC with identifier 'unitySplashStoryboard'

### ISSUE THREAD

**Issue by Steven4294** [jiulongw/swift-unity#71]
When I run the demo, after building the unity project and running the xcode project I launch the app with the crash:

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Storyboard (<UIStoryboard: 0x1d0075740>) doesn't contain a view controller with identifier 'unitySplashStoryboard''
*** First throw call stack:
(0x185017164 0x184260528 0x18eee8fb8 0x10559de5c 0x10559f488 0x104e62784 0x1055bf14c 0x1055bf790 0x18e6402c4 0x18e831bc4 0x18e836a14 0x18ead22a8 0x18edaa100 0x18ead1f0c 0x18ead276c 0x18f2577c0 0x18f257664 0x18efc52cc 0x18f15e3cc 0x18efc517c 0x18eda9760 0x18e835158 0x18ec4adbc 0x1876da1f0 0x1876e2af8 0x109fed28c 0x109ff99e4 0x18770e7f8 0x18770e49c 0x18770ea38 0x184fbf77c 0x184fbf6fc 0x184fbef84 0x184fbcb5c 0x184edcc58 0x186d88f84 0x18e6355c4 0x1055c0598 0x1849fc56c)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

Xcode is version 9.2
Unity is version 2017.4.2f2

**Comment by jiulongw** [comment]
2017.4.2f2 is not supported yet...

**Comment by jiulongw** [comment]
Fixed in 1b3283bfaa71a594765f07a97b1adddc7d99971e
